chain/chain/src/chain.rs

```diff
@@ -1923,19 +1923,45 @@ impl Chain {
 
         if let Some(tip) = &new_head {
             // TODO: move this logic of tracking validators metrics to EpochManager
-            let mut count = 0;
+            let mut chunk_producers_count = 0;
+            let mut block_producers_count = 0;
+            let mut chunk_validators_count = 0;
```

Comment:
small nit: Can you change the order from most important to least important? It should be BP, CP, CV. Same in other places.